CMakeLists.txt

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # SPDX-License-Identifier: MIT
 
-cmake_minimum_required (VERSION 3.12)
+cmake_minimum_required (VERSION 3.15)
```

Comment:
The justification of upgrade this version?

Reply:
i'm gonna look into it and see if it's necessary to upgrade.

Reply:
See https://teams.microsoft.com/l/message/19:a5180a2de11a40cb83a822f588d60c15@thread.skype/1592287299673?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1592287299673&teamName=Azure%20SDK&channelName=Language%20-%20C%E1%90%A9%E1%90%A9&createdTime=1592287299673

Reply:
Which platforms are you targeting? I moved one of my internal projects to 3.15 and am discovering that Ubuntu 18.04 LTS is 3.10.